stable/nats/templates/ingress.yaml

```diff
@@ -0,0 +1,36 @@
+{{- if .Values.ingress.enabled -}}
+{{- range .Values.ingress.hosts }}
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: {{ template "nats.fullname" $ }}-monitoring
+  labels:
+    app: "{{ template "nats.name" $ }}"
+    chart: "{{ $.Chart.Name }}-{{ $.Chart.Version }}"
```

Comment:
use the template here: `"{{ template "nats.chart" . }}"`

Reply:
👍